types/meyda/index.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for meyda 4.2
+// Project: https://github.com/meyda/meyda
+// Definitions by: Damien Erambert <https://github.com/eramdam>
+//                 Hugh Rawlinson <https://github.com/hughrawlinson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type MeydaWindowingFunction = 'blackman' | 'sine' | 'hanning' | 'hamming';
+export type MeydaAudioFeature =
+    | 'amplitudeSpectrum'
+    | 'chroma'
+    | 'complexSpectrum'
+    | 'energy'
+    | 'loudness'
+    | 'mfcc'
+    | 'perceptualSharpness'
+    | 'perceptualSpread'
+    | 'powerSpectrum'
+    | 'rms'
+    | 'spectralCentroid'
+    | 'spectralFlatness'
+    | 'spectralFlux'
+    | 'spectralKurtosis'
+    | 'spectralRolloff'
+    | 'spectralSkewness'
+    | 'spectralSlope'
+    | 'spectralSpread'
+    | 'zcr'
+    | 'buffer';
+
+export type MeydaSignal = SliceableArrayLike<number>;
+export interface SliceableArrayLike<T> extends ArrayLike<T> {
+    slice(start: number, end: number): SliceableArrayLike<T>;
+}
+
+type MeydaSource = AudioNode;
+
+export interface MeydaFeaturesObject {
+    amplitudeSpectrum: Float32Array;
+    buffer: number[];
+    chroma: number[];
+    complexSpectrum: {
+        real: number[];
+        imag: number[];
+    };
+    energy: number;
+    loudness: {
+        specific: Float32Array;
+        total: number;
+    };
+    mfcc: number[];
+    perceptualSharpness: number;
+    perceptualSpread: number;
+    powerSpectrum: Float32Array;
+    rms: number;
+    spectralCentroid: number;
+    spectralFlatness: number;
+    spectralKurtosis: number;
+    spectralRolloff: number;
+    spectralSkewness: number;
+    spectralSlope: number;
+    spectralSpread: number;
+    zcr: number;
+}
+
+export class MeydaAnalyzer {
+    start(features: MeydaAudioFeature[]): void;
+
+    stop(): void;
+
+    setSource(source: MeydaSource): void;
+
+    get(features: MeydaAudioFeature[]): Partial<MeydaFeaturesObject> | null;
+}
+
+interface Meyda {
+    audioContext: AudioContext | null;
+    spn: ScriptProcessorNode | null;
+    bufferSize: number;
+    sampleRate: number;
+    melBands: number;
+    chromaBands: number;
+    callback: (features: Partial<MeydaFeaturesObject>) => void | null;
+    windowingFunction: MeydaWindowingFunction;
+    featureExtractors: any;
+    EXTRACTION_STARTED: boolean;
+    numberOfMFCCCoefficients: number;
+    windowing(signal: MeydaSignal, windowname?: MeydaWindowingFunction): MeydaSignal;
+
+    createMeydaAnalyzer(options: MeydaAnalyzerOptions): MeydaAnalyzer;
+    extract(
+        feature: MeydaAudioFeature | MeydaAudioFeature[],
+        signal: MeydaSignal,
+        previousSignal?: MeydaSignal,
+    ): Partial<MeydaFeaturesObject> | null;
+}
+
+export const main: Meyda;
+
+export default main;
```

Comment:
I did a quick check on the library and I didn't found where you export a default. I think this is incorrect.